packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -17,6 +17,16 @@
 
   /* Having a z-index at least "wins" by default */
   z-index: 1;
+
+  /* top-align at the top 20% of the screen */
+  padding-top: 20vh;
```

Comment:
add `box-sizing: border-box;` as well, otherwise this is adding 20vh to the total element height, making it taller than the viewport.



![Screen Shot 2021-12-01 at 11 10 20 AM](https://user-images.githubusercontent.com/6165315/144299239-89fbb73e-8ea7-4fbd-b662-dbc573d6f7f2.png)
![2021-12-01 11 07 15](https://user-images.githubusercontent.com/6165315/144299284-84c9251d-513a-4fbd-abfe-295f2040eeff.gif)

Reply:
Good catch, adjusting:
```suggestion
  box-sizing: border-box; // don't add padding to total height
  padding-top: 20vh;
```